recipes/pycoast/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pycoast" %}
+{% set version = "1.1.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "3359b0cc977b13b4b96628a7c2884e85548a08e57220d78734632f914ed1246c" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - pyshp
+    - numpy
+    - pyproj
+    - pillow
+    - six
+    - aggdraw
+  run:
+    - python
+    - pyshp
+    - numpy
+    - pyproj
+    - pillow
+    - six
+    - aggdraw
+
+test:
+  imports:
+    - pycoast
+    - pycoast.tests
+
+about:
+  home: https://github.com/pytroll/pycoast
+  license: GPLv3
```

Comment:
`license: GPL-3.0`

https://opensource.org/licenses/alphabetical

Reply:
@synapticarbors This naming was taken as part of the output from `conda skeleton`. Similarly the `host` requirements were from that too, I'll fix both I guess.

Reply:
@synapticarbors FYI I was adding a license file to a different project and found this page for GPL talking about specifying future license versions: https://www.gnu.org/licenses/identify-licenses-clearly.html

Not sure if this affects the type of metadata you brought up here, but wanted to mention it.